pkg/ui/console/client.go

```diff
@@ -110,6 +110,15 @@ func CreateClientFromConfig(ctx context.Context, cfg *config.Config) (*Client, e
 		if err != nil {
 			return nil, err
 		}
+		if ok, err := dialect.Validate(ctx); err != nil {
+			if !ok {
+				return nil, fmt.Errorf("validate: %w", err)
+			}
+			log.Warn().Err(err).Msg("database validation warning")
```

Comment:
Do we want a better message?

Reply:
the `err` is either:
`unsupported PostgreSQL version: %s. (should be >= %s)`
or
`error getting PostgreSQL version: %w`